gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Monkey.groovy

```diff
@@ -22,4 +26,10 @@ class Monkey extends AdbTask {
             events = events.call()
         events ?: pluginEx.events
     }
+
+    private getSeed() {
+        if (seed instanceof Closure)
+            seed = seed.call()
+        seed ?: pluginEx.getSeed()
```

Comment:
Might use `pluginEx.seed` as this is the standard property access. It results in the getter being called.